clients/shared_library/components/pages/CoursePhaseParticpationsTable/CoursePhaseParticipationsTablePage.tsx

```
@@ -126,7 +128,11 @@ export const CoursePhaseParticipationsTablePage = ({
             <TableBody>
               {table.getRowModel().rows?.length ? (
                 table.getRowModel().rows.map((row) => (
-                  <TableRow key={row.id} data-state={row.getIsSelected() && 'selected'}>
+                  <TableRow
+                    key={row.id}
+                    data-state={row.getIsSelected() && 'selected'}
+                    onClick={() => onClickRowAction && onClickRowAction(row.original)}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)